.github/workflows/release.yaml

```diff
@@ -54,13 +58,32 @@ jobs:
           RELEASE: ${{ env.RELEASE_VERSION }}-ARM
         run: |
           cd opensearch-operator
-          pwd
-          ls
-          go version
-          go env
           aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/opsterio
           docker build -f Dockerfile-ARM -t public.ecr.aws/opsterio/opensearch-operator:$RELEASE ./
           docker push public.ecr.aws/opsterio/opensearch-operator:$RELEASE
           docker tag public.ecr.aws/opsterio/opensearch-operator:$RELEASE public.ecr.aws/opsterio/opensearch-operator:latest-ARM
           docker push public.ecr.aws/opsterio/opensearch-operator:latest-ARM
-
+      - name: Copy-crd-to-chart-dir
+        run: |
+          cd opensearch-operator
+          cp and mv can be done in one call:
```

Comment:
Line should not be in here. Copy-paste-error?

Reply:
Line is still here

Reply:
Hey @swoehrl-mw, thanks for your review !
i think you right, the version and app version should be on the release workflow and the tag on the helm-release.
correct me if i im wrong :)